app/controllers/v3/droplets_controller.rb

```diff
@@ -120,6 +124,7 @@ def upload
     droplet_not_found! unless droplet && permission_queryer.can_read_from_space?(droplet.space.guid, droplet.space.organization.guid)
 
     unauthorized! unless permission_queryer.can_write_to_space?(droplet.space.guid)
+    suspended! unless permission_queryer.is_org_active?(droplet.space.organization.guid)
```

Comment:
Add tests to `spec/request/droplets_spec.rb`:
- POST /v3/droplets
- POST /v3/droplets?source_guid=:guid
- POST /v3/droplets/:guid/upload
- PATCH v3/droplets/:guid
- DELETE /v3/droplets/:guid

Reply:
Done